request-vm-cnv/application.properties

```diff
@@ -1,6 +1,6 @@
 # Backstage Notifications service
 quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator}
-quarkus.openapi-generator.notifications.auth.bearertoken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN}
+quarkus.openapi-generator.notifications.auth.BearerToken.bearer-token=${NOTIFICATIONS_BEARER_TOKEN}
```

Comment:
not sure if case sensitive, did you test?
But better safe than sorry on this one :)

Reply:
it fails when there is a mismatch

Reply:
thx